External/HeCBench/README

```diff
@@ -5,6 +5,19 @@ This directory contains a CMakeLists.txt for the HeCBench so it can be built as
 of the LLVM test-suite. Its sources are not part of the test-suite but
 have to be fetched separately from https://github.com/zjin-lcf/HeCBench.
 
+The CMakeLists.txt contains internal greenlists and redlists that the user
+must declare which options to use with SYSTEM_GPU. The user is able to
+declare multiple lists to use.
+
+the naming convention for all of the options is gpu/benchmarks followed by
+the expected outcome.
+
+here is am example for all of the options for amd:
```

Comment:
can you say how you can do it for nvidia, something similar to: these are the options and have <amd/nvidia>